src/guide/project-maintenance.md

```diff
@@ -0,0 +1,66 @@
+# Project Maintenance
+
+Many OpenJDK projects build on top of the JDK source code. When doing this there are a number of common workflows that are dealt with by most project maintainers. For instance updating the code (merging) to bring in the latest changes from the upstream JDK project.
+
+## Merging JDK mainline into a project repository
+
+Merging changes from one git repository to another is basically the same thing as getting your own changes merged into the project repository, with the slight twist that you don't write all the changes yourself, you just pull them from somewhere else.
+
+In this example we'll use a separate clone of the project repository to perform the merge in. This can be done using branches as well, but let's keep it simple for now.
+
+### Init - done once
+
+First set up your personal fork of the project repository, in this example called `my-project`. If you already are a contributor to the project you most likely have this set up. If not, see [Cloning the JDK](#cloning-the-jdk) for details on how to do that.
+
+~~~
+git clone git@github.com:OpenDuke/my-project.git project-merge
```

Comment:
People may not realise that `OpenJDK` is a username here.

Reply:
Clarified.